src/main/java/org/mockbukkit/mockbukkit/inventory/RecipeManager.java

```diff
@@ -29,12 +34,34 @@ public class RecipeManager
 {
 
 	private static final String RECIPE_TYPE_CANNOT_BE_NULL = "Recipe type cannot be null";
+	private static final String THE_RECIPE_CANNOT_BE_NULL = "The recipe cannot be null";
+	private static final String THE_CRAFTING_MATRIX_CANNOT_BE_NULL = "The craftingMatrix cannot be null";
```

Comment:
Personal opinion here, up for discussion. Is it necessary to not replicate exception messages. Like the content of those messages does not really need to match each other. I know the linter complains about this, but what does really show most clarity in the code?

Reply:
Yes, i agree with you. This is a rule that's a little bit annoying. Would be nice to have a way to disable this rule.

Reply:
I will look into it